storybook/storybook-react/config/preview.tsx

```diff
@@ -11,12 +13,22 @@ export const decorators = [
       <Story />
     </div>
   ),
+  withThemeByClassName({
+    themes: {
+      spacious: 'amsterdam-theme--spacious',
+      compact: 'amsterdam-theme--compact',
+    },
+    defaultTheme: 'spacious',
```

Comment:
This looks a little bit better in the button.

```suggestion
    themes: {
      Spacious: 'amsterdam-theme--spacious',
      Compact: 'amsterdam-theme--compact',
    },
    defaultTheme: 'Spacious',
```